eth/handler.go

```diff
@@ -88,6 +89,8 @@ type ProtocolManager struct {
 	txsSub        event.Subscription
 	minedBlockSub *event.TypeMuxSubscription
 
+	whitelist *common.BlockHashWhitelist
```

Comment:
There's no reason to use a pointer here. A map can be nil.

Reply:
Done.